django/db/models/options.py

```diff
@@ -193,6 +193,18 @@ def contribute_to_class(self, cls, name):
             self.unique_together = normalize_together(self.unique_together)
             self.index_together = normalize_together(self.index_together)
 
+            # check for the %(class)s escape used for inherited permissions.
+            # If present, replace it with the appropriate text based off of
+            # the class name for both the name and codename of the permission.
+            perms = meta_attrs.pop('permissions', self.permissions)
+            translated_perms = []
```

Comment:
I'd call the variable `interpolated_perms` -- "translated" gives the idea that some translation happens.